### ISSUE TITLE
React 18 and Tweens

### ISSUE THREAD

**Issue by paulieDOTjs** [bitworking/react-gsap#69]
From animations do not work anymore after updating to React 18.

**Comment by nomadoda** [comment]
Details here: https://greensock.com/forums/topic/31712-simple-opacity-fade-doesnt-work-in-react/#comment-158550

**Comment by bitworking** [comment]
Sorry, I was busy with other things the last months. Hmm, I just updated every package locally and tried it with React 18 and Strict Mode in the local Next.js. It runs without problems on my side. I have to do some other tests.

Does it work for you if you disable Strict Mode?

**Comment by bitworking** [comment]
Ok, I see it now. I first didn't saw that it's only not working for "from" animations. Will have a look at this, thank you.

**Comment by bitworking** [comment]
A quick fix would be to not use the Strict Mode: https://stackblitz.com/edit/react-ts-fmarqq?file=App.tsx

**Comment by paulieDOTjs** [comment]
Yes, that would be a quick fix, but there are instances where one would want to use Strict Mode
